driver-core/src/main/com/mongodb/client/model/UpdateOptions.java

```diff
@@ -256,6 +258,32 @@ public UpdateOptions let(final Bson variables) {
         return this;
     }
 
+    /**
+     * Gets the sort criteria to apply to the query. The default is null, which means that the documents will be returned in an undefined
+     * order.
+     *
+     * @return a document describing the sort criteria
+     * @since 5.3
+     * @mongodb.driver.manual reference/method/cursor.sort/ Sort
+     */
+    @Nullable
+    public BsonDocument getSort() {
+        return sort;
+    }
+
+    /**
+     * Sets the sort criteria to apply to the query.
+     *
```

Comment:
Could we clarify the purpose of this sort option in the Javadoc? It would be helpful to explain how it determines which document is updated when multiple matches are found.

```suggestion
     *
     * When multiple documents match the query, the sort order specifies the document 
     * to be updated first.
     *
```

Reply:
Will do!